.changeset/lemon-onions-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@redocly/openapi-core": minor
+---
+
+Added new `missing-required-schema-properties` rule to check if any of required schema properties is missing.
```

Comment:
```suggestion
Added new `no-required-schema-properties-undefined` rule to check if each required schema properties is defined.
```
To be consistent with our naming, it should be named like `no-xxxxxxxx`. I proposed `no-required-schema-properties-undefined`.

Reply:
Fixed as well!